tools/find_dart.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I'd skip the shebang business since it won't work on Windows anyway, afaik.

If a wrapper needs to add some engine repo specific setup or logic, then it would go under `bin/dart(.bat)`, similar to the file at the same location in the framework repo.